cmd/create/cluster/cmd.go

```diff
@@ -359,6 +359,20 @@ func run(cmd *cobra.Command, _ []string) {
 	}()
 	ocmClient := ocmConnection.ClustersMgmt().V1()
 
+	tempAWSClient := aws.GetAWSClientForUserRegion(reporter, logger)
+
+	creator, err := tempAWSClient.GetCreator()
+	if err != nil {
+		reporter.Errorf("Unable to get IAM credentials: %v", err)
+	}
+
+	// Creating non STS clusters is not supported
+	if creator.IsSTS && args.roleARN == "" {
+		reporter.Errorf("IAM STS credentials can only create STS clusters --role-arn cannot be blank")
+		reporter.Errorf("You are AWS credentials returning an STS ARN: %s please create an STS cluster", creator.ARN)
```

Comment:
Change to `Your AWS credentials are returning an STS ARN: '%s'.` (Remove the `please...` part)

Reply:
Fixed.